### PR TITLE
Add a note to dotnet nuget add source

### PR DESCRIPTION
**Pull request by Youssef1313**
FYI @gewarren

cc @mmitche if you want to add more details.

### CODE REVIEW

File: docs/core/tools/dotnet-nuget-add-source.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ dotnet nuget add source -h|--help
 
 The `dotnet nuget add source` command adds a new package source to your NuGet configuration files.
 
+> [IMPORTANT]
+> Avoid having the public `nuget.org` feed along with other feeds. For more information, see [Dependency confusion attack](https://www.msn.com/money/other/microsoft-warns-enterprises-of-new-dependency-confusion-attack-technique/ar-BB1dynJl).
```
Comment (IEvangelist) [inline docs/core/tools/dotnet-nuget-add-source.md:29]: This link is invalid, doesn't resolve for me. I did find related articles, I'm not sure this is directly related though.

**Review by IEvangelist — CHANGES_REQUESTED** [review]
I'd like to have @blowdart chime in here for the correct course of action - thank you.

I wonder if linking to something like this is more appropriate? https://docs.microsoft.com/nuget/concepts/security-best-practices

File: docs/core/tools/dotnet-nuget-add-source.md (hunk 1 above)
Comment (Youssef1313) [reply docs/core/tools/dotnet-nuget-add-source.md]: @IEvangelist See also https://github.com/dotnet/docs/pull/22777.

**Comment by blowdart** [comment]
The intent is good but linking to news articles is bad. Instead, consider something like, wrapped in the warning chrome?

Care should be taken when adding multiple feeds not to introduce a [dependency confusion attack](https://aka.ms/pkg-sec-wp).

**Comment by blowdart** [comment]
That works, I leave it to @IEvangelist to chime in on wording :)

File: docs/core/tools/dotnet-nuget-add-source.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ dotnet nuget add source -h|--help
 
 The `dotnet nuget add source` command adds a new package source to your NuGet configuration files.
 
+> [!WARNING]
+> Be careful when adding multiple feeds not to introduce a [dependency confusion attack](https://aka.ms/pkg-sec-wp).
```
Comment (IEvangelist) [inline docs/core/tools/dotnet-nuget-add-source.md:29]:
```suggestion
> [!WARNING]
> When adding multiple NuGet sources, be careful not to introduce a [dependency confusion attack](https://aka.ms/pkg-sec-wp).
```

**Review by IEvangelist — APPROVED** [review]

File: docs/core/tools/dotnet-nuget-add-source.md (hunk 2 above)
Comment (Youssef1313) [reply docs/core/tools/dotnet-nuget-add-source.md:29]: @IEvangelist Should it be multiple package sources instead to match the name used in [nuget.config](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesources)?

Comment (IEvangelist) [reply docs/core/tools/dotnet-nuget-add-source.md:29]: Yeah, that is fine. I updated the suggestion...

**Review by mmitche — APPROVED** [review]

File: docs/core/tools/dotnet-nuget-add-source.md [hunk 3]
```diff
@@ -25,6 +25,9 @@ dotnet nuget add source -h|--help
 
 The `dotnet nuget add source` command adds a new package source to your NuGet configuration files.
 
+> [!WARNING]
+> When adding multiple package sources, be careful not to introduce a [dependency confusion attack](https://aka.ms/pkg-sec-wp).
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-add-source.md:29]:
```suggestion
> When adding multiple package sources, be careful not to introduce a [dependency confusion vulnerability](https://aka.ms/pkg-sec-wp).
```

**Review by tdykstra — APPROVED** [review]
Thanks, @Youssef1313 !

**Pull request merged**